webapp/i18n/es.json

```diff
@@ -10,5 +10,9 @@
   "sidebar.enabled": "Habilitado",
   "tooltip.message": "Esta es una información sobre herramientas personalizada del complemento de demostración",
   "useractions.action": "Acción",
-  "useractions.demo": "Complemento de Demostración: "
+  "useractions.demo": "Complemento de Demostración: ",
+  "submenu.menu": "Ejemplo de submenú",
+  "submenu.first": "Primer iteam",
```

Comment:
`item` not `iteam` but probably it should sound more natural in spanish like `elemento`.

Reply:
Done